project-koku/koku
# [COST-4741] Add UsageType and Operation to daily data

## Jira Ticket

[COST-3761](https://issues.redhat.com/browse/COST-3761)
[COST-4741](https://issues.redhat.com/browse/COST-4741)

## Description

Add additional columns to the AWS parquet files needed to determine node network costs.

## Testing

1. Checkout Branch
2. Restart Koku
3. Drop external tables

```
python koku/manage.py migrate_trino_tables --drop-tables aws_line_items,aws_line_items_daily,aws_openshift_daily
```

4. Download data so that the external tables are recreated
5. Check that the columns exist with expected data

```
trino:org1234567> SELECT DISTINCT lineitem_usagetype,lineitem_operation from aws_line_items;
```

## Release Notes

```markdown
* [COST-4741](https://issues.redhat.com/browse/COST-4741) Add AWS usage type and operation columns to the Trino tables
```
